src/thunderbots/software/radio_communication/mrf/dongle.cpp

```diff
@@ -0,0 +1,610 @@
+#include "dongle.h"
+
+#include <sigc++/bind.h>
+#include <sigc++/functors/mem_fun.h>
+#include <sigc++/reference_wrapper.h>
+#include <unistd.h>
+
+#include <algorithm>
+#include <bitset>
+#include <cassert>
+#include <chrono>
+#include <cstdlib>
+#include <cstring>
+#include <exception>
+#include <iostream>
+#include <stdexcept>
+#include <string>
+#include <unordered_map>
+
+#include "constants.h"
+#include "radio_communication/visitor/mrf_primitive_visitor.h"
+#include "util/logger/init.h"
+
+namespace
+{
+    struct RadioConfig
+    {
+        uint8_t channel;
+        int symbol_rate;
+        uint16_t pan;
+    };
+
+    const RadioConfig DEFAULT_CONFIGS[4] = {
+        {25U, 250, 0x1846U},
+        {25U, 250, 0x1847U},
+        {25U, 250, 0x1848U},
+        {25U, 250, 0x1849U},
+    };
+
+    const unsigned int ANNUNCIATOR_BEEP_LENGTH = 750;
+
+    static const uint64_t MAC = UINT64_C(0x20cb13bd834ab817);
```

Comment:
What's this MAC for? The dongle?

Reply:
probably?

Reply:
Lol, could we leave a quick comment saying that this is probably for the dongle, just to give people in the future _some_ sort of direction.

Reply:
Yea not exactly sure what that is either. MAC of the dongle seems like the best guess. I can't find that value in our udev rules file, and it only shows up once in the `dongle.cpp` class in the old code. We might be able to check with the electrical team to confirm this.

Reply:
Confirmed with Logan that this is (99.99999%) the dongle MAC